.aspect/workflows/README.md

```diff
@@ -0,0 +1,66 @@
+# Aspect Workflows demonstration deployment
+
+This deployment of [Aspect Workflows](https://www.aspect.build/workflows) is configured to run on AWS + Buildkite.
+
+You can see this Aspect Workflows demonstration deployment live at
+https://buildkite.com/aspect/rules-jest.
+
+The three components of the configuration are,
+
+1. Aspect Workflows terraform module
+1. Aspect Workflows configuration yaml
+1. Buildkite pipeline configuration (in the Buildkite UI)
+
+## Aspect Workflows terraform module
+
+This is found under the [.aspect/workflows/terraform](./terraform) directory.
+
+## Aspect Workflows configuration yaml
+
+This is the [config.yaml](./config.yaml) file in this directory.
+
+## Buildkite pipeline configuration (in the Buildkite UI)
+
+There are two pipelines configured on Buildkite.
+
+1. Main build & test pipeline: https://buildkite.com/aspect/rules-jest
+2. Scheduled warming pipeline: https://buildkite.com/aspect/rules-jest-warming
```

Comment:
Perhaps another sentence describing what warming is?

Reply:
Oh I see you have a blurb below. I might add slightly more detail like ..."caches repository rules so that runners don't have to re-fetch them".

Reply:
👍